actors.md

```diff
@@ -4,6 +4,42 @@ Any implementations of the Filecoin actors must be exactly byte for byte compati
 
 This spec decsribes a set of actors that operate within the [Filecoin State Machine](state-machine.md). All types are defined in [the basic type encoding spec](data-structures.md#basic-type-encodings).
 
+## Init Actor
+The init actor is responsible for creating new actors on the filecoin network. This is a built-in actor and cannot be replicated. In the future, this actor will be responsible for loading new code into the system (for user programmable actors).
+
+```go
+type InitActor struct {
+    NextActorID uint64
+}
+```
+
+#### Exec
+
+Parameters:
+
+- code Cid
+- params []byte
+
+TODO: the type of params should probably just be variadic set of arguments. Making it be a `[]byte` here kinda sucks
+
+```go
+func Exec(code Cid, params []byte) Address {
+    actorID := self.NextActorID
+    self.NextActorID++
+    
+    if code == InitActorCodeCid {
```

Comment:
I think this should be a white list for now, given there are many actors we don't want other to launch

Reply:
I actually don't mind people launching their own storage market with its own miners. I think that would be interesting. There could be other storage markets for users who don't want to participate in block production, or something.

Reply:
If we allow that, then we should have a way to ask the chain directly for the official versions, to reduce chances that anyone is scammed/or accidentally uses a wrong version

Reply:
yeah... maybe it makes the most sense to not allow other storage markets for now then too. Can always re-enable it later

Reply:
done